src/styles/sass_style/block.scss

```
@@ -0,0 +1,101 @@
+@import "sass_style/variable.scss";
+
+.stars {
+  display: flex;
+  align-items: center;
+  &__star {
```

```suggestion

  &__star {
```
Change all similar places adding whitespaces [example](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)